modules/ROOT/partials/configuration/comments-tinycomments_access.adoc

```diff
@@ -0,0 +1,63 @@
+[[tinycomments-access]]
+== `tinycomments_access`
+
+*Type:* `+String+`
+
+*Possible values:* `'comment'`
+
+This option controls the editor's read-only state and comment functionality.
+
+[IMPORTANT]
+====
+* When `tinycomments_access` is set to `comment`, the editor will automatically switch to xref:editor-important-options.adoc#readonly[Readonly Mode].
+* Default behavior, when the `tinycomments_access` option is either not configured or set to an empty string, allows full editing access to both the editor and the comments.
+====
+
+=== `tinycomments_access` _is set to `'comment'` and `readonly` set to `true`_
+
+When `tinycomments_access` is set to `comment` and `readonly` is set to `true`, the editor **will remain** in read-only mode, but the comments will be in **comment-only** mode. This means that users can add and delete comments, but they cannot edit the editor content.
+
+[cols="1,1,2", options="header"]
+|===
+|Editor Readonly |Tinycomments_access |Result
+|true |comment |The editor is read-only, but users can add or delete new comments.
+|false |comment |The editor is read-only, but users can add or delete new comments.
+|===
+
+.Example
+[source,javascript]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'tinycomments',
+  toolbar: 'addcomment showcomments',
+  readony: true, // set the editor to read-only mode
```

Comment:
```suggestion
```
Unnecessary, `tinycomments_access: "comment"` sets the editor to readonly already, so the integrator doesn't need to use the `readonly` option.